scripts/irods/test/test_igroupadmin.py

```diff
@@ -149,13 +149,15 @@ def test_groupadmin_can_lock_self_out_of_group__issue_7707(self):
             self.groupadmin.assert_icommand(['igroupadmin', 'mkgroup', self.group])
             self.groupadmin.assert_icommand(['igroupadmin', 'mkuser', test_username])
 
-            # Populate empty group. groupadmin should no longer have any control of this group now
+            # Populate empty group. groupadmin should no longer have any control of the group now
             self.groupadmin.assert_icommand(['igroupadmin', 'atg', self.group, test_username])
 
-            # Assert groupadmin cannot control this group now
-            self.groupadmin.assert_icommand_fail(['igroupadmin', 'atg', self.group, self.groupadmin.username])
-            self.groupadmin.assert_icommand_fail(['igroupadmin', 'rfg', self.group, test_username])
-            self.groupadmin.assert_icommand_fail(['igroupadmin', 'lg', self.group])
+            # Assert groupadmin cannot control the group
+            self.groupadmin.assert_icommand(['igroupadmin', 'atg', self.group, self.groupadmin.username], 'STDERR', 'CAT_INSUFFICIENT_PRIVILEGE_LEVEL', desired_rc=0)
+            self.groupadmin.assert_icommand(['igroupadmin', 'rfg', self.group, test_username], 'STDERR', 'CAT_INSUFFICIENT_PRIVILEGE_LEVEL', desired_rc=0)
```

Comment:
To future proof this even more, include the iRODS error code with the symbolic name. That way, if `<prefix>CAT_INSUFFICIENT_PRIVILEGE_LEVEL<suffix>` is returned, the test won't be tripped up by it.

Reply:
Done.